proto/scanner/api/v1/note.proto

```diff
@@ -12,3 +12,8 @@ enum Note {
   LANGUAGE_CVES_UNAVAILABLE       = 2;
   CERTIFIED_RHEL_SCAN_UNAVAILABLE = 3;
 }
+
+enum NodeNote {
+  NODE_OS_UNSUPPORTED     = 0;
```

Comment:
OS unsupported and Kernel unsupported may be confusing. Going to rename `NODE_OS_UNSUPPORTED` to just `NODE_UNSUPPORTED`